node-operators/networks/collator.md

```diff
@@ -82,3 +89,77 @@ The following table presents some of the timings in regards to different actions
 !!! note 
     The values presented in the previous table are subject to change in future releases.
 
+## Session Keys
+
+With the release of [Moonbase Alpha v8](/networks/testnet/), collators will sign blocks using an author ID, which is basically a [session key](https://wiki.polkadot.network/docs/en/learn-keys#session-keys). To match the Substrate standard, Moonbeam collator's session keys are [SR25519](https://wiki.polkadot.network/docs/en/learn-keys#what-is-sr25519-and-where-did-it-come-from). This guide will show you how you can create/rotate your session keys associated to your collator node.
+
+First, make sure you are running your collator node with the following flags. TODO: Art make sure the flags are included in the collator's code snippet, also. If there are some flags that add some security concerns, it might be good to point them out.
```

Comment:
@artkaseman looks like some flags need to be added here

Reply:
fixed